component/acme-dns.libsonnet

```diff
@@ -137,8 +137,9 @@ local scriptPodSpec(name, script) = {
     },
     scripts: {
       configMap: {
-        // 0700
-        defaultMode: 448,
+        // We need mode 0770 for the scripts, so they work correctly on OCP
+        // 4.11 where the configmap contents are owned by root:<pod-GID>.
+        defaultMode: 504,
```

Comment:
I was very confused at first because decimal != octal :sweat_smile:   Any reason we don't use `std.parseOctal`?

Reply:
I probably just didn't remember that `std.parseOctal()` exists, I'll adjust this PR to switch to using that.